### PR TITLE
Search input component

### PR DESCRIPTION
**Pull request by fares98**
Create a search input component and use the Button component

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -44,4 +44,4 @@
       "last 1 safari version"
     ]
   }
-}
+}
```
Comment (mossa-Sammer) [inline client/package.json]: add a blank line at the end of the file

File: client/src/components/common/SearchInput/style.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  left: -4px;
+  background: var(--main-color);
+  color: var(--secondery-color);
+  font-size: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+
```
Comment (mossa-Sammer) [inline client/src/components/common/SearchInput/style.css]: leave one blank line only at the end of the file

File: client/src/components/common/SearchInput/style.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+.search-input{
+  width: 85%;
+  border-radius: 8px;
+  font-size: 20px;
+  padding: 5px 15px;
+  border: transparent;
+  background-color: var(--secondery-color);
+}
+
+
```
Comment (mossa-Sammer) [inline client/src/components/common/SearchInput/style.css]: remove this blank line

File: client/src/components/common/SearchInput/style.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
+.form{
```
Comment (AbdallahAmmar96) [inline client/src/components/common/SearchInput/style.css]: this class is too common

File: client/src/components/common/SearchInput/style.css (hunk 3 above)
Comment (AbdallahAmmar96) [inline client/src/components/common/SearchInput/style.css]: delete this line

File: client/src/components/common/SearchInput/style.css (hunk 2 above)
Comment (AbdallahAmmar96) [inline client/src/components/common/SearchInput/style.css]: delete this line

**Pull request merged**